edb/pgsql/keywords.py

```diff
@@ -68,7 +68,7 @@
     "cast": ("CAST", RESERVED_KEYWORD),
     "catalog": ("CATALOG_P", UNRESERVED_KEYWORD),
     "chain": ("CHAIN", UNRESERVED_KEYWORD),
-    "char": ("CHAR_P", COL_NAME_KEYWORD),
+    "char": ("CHAR_P", TYPE_FUNC_NAME_KEYWORD),
```

Comment:
I'm not quite sure this is the best fix. Maybe we should special-case `"char"` in codegen instead?

Reply:
Maybe here?

https://github.com/edgedb/edgedb/blob/master/edb/pgsql/common.py#L105-L131

Reply:
No, I think none of that actually solves the confusion.  IMO, the most reliable way would be to translate unquoted `char` to `bpchar` at the parser level and always quote `"char"` in codegen.  In other words, use Postgres names.